website/docs/tutorials/04-configuration-file.md

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_label: Configuration file
+---
```

Comment:
```suggestion
```

nitpick: this is not needed because Docusaurus can pick this up from page title automatically

Reply:
Removed